src/components/settings/Account/Billing/alerts/ServiceCutoffAlert.tsx

```diff
@@ -0,0 +1,148 @@
+import { AccountStatus, useAccount } from "@3rdweb-sdk/react/hooks/useApi";
+import {
+  useDisclosure,
+  Alert,
+  Flex,
+  AlertIcon,
+  AlertDescription,
+  IconButton,
+  AlertTitle,
+  UnorderedList,
+} from "@chakra-ui/react";
+import { OnboardingBilling } from "components/onboarding/Billing";
+import { OnboardingModal } from "components/onboarding/Modal";
+import { useState } from "react";
+import { FiX } from "react-icons/fi";
+import { Heading, Text, TrackedLinkButton } from "tw-components";
+import { ManageBillingButton } from "../ManageButton";
+import { getRecurringPaymentFailureResponse } from "lib/billing";
+
+type ServiceCutoffAlertProps = {
+  onDismiss?: () => void;
+  affectedServices?: string[];
+  paymentFailureCode: string;
+};
+
+export const ServiceCutoffAlert: React.FC<ServiceCutoffAlertProps> = ({
+  onDismiss,
+  affectedServices = [],
+  paymentFailureCode = "bank_decline",
+}) => {
+  // TODO: We should find a way to move this deeper into the
+  // TODO: ManageBillingButton component and set an optional field to override
+  const [paymentMethodSaving, setPaymentMethodSaving] = useState(false);
```

Comment:
`useState<boolean>`

Reply:
not required, `false` is inferred to `boolean` correctly so does not require the annotation